ext/gd/gd.c

```diff
@@ -3545,7 +3545,7 @@ PHP_FUNCTION(imageflip)
 			break;
 
 		default:
-			zend_argument_value_error(2, "must be a valid mode");
+			zend_argument_value_error(2, "must be either GD_FLIP_VERTICAL, GD_FLIP_HORINZONTAL, or GD_FLIP_BOTH");
```

Comment:
Typo here. HORINZONTAL should be HORIZONTAL.

Reply:
You've just spotted it just in time! I'll fix it, thanks!

Reply:
Oh, wait! The (internal) constant itself also has the same issue! I'll fix that as well.